Prevents:
error MSB3073: The command "git describe --tags --always --dirty" exited with code 128
when building from tar archive instead of git repository

--- a/Marksman/Marksman.fsproj
+++ b/Marksman/Marksman.fsproj
@@ -4,6 +4,7 @@
         <OutputType>Exe</OutputType>
         <AssemblyName>marksman</AssemblyName>
         <VersionPrefix>1.0.0</VersionPrefix>
+        <VersionString>@TERMUX_PKG_VERSION@</VersionString>
         <TargetFramework>net9.0</TargetFramework>
         <IncludeSourceRevisionInInformationalVersion>false</IncludeSourceRevisionInInformationalVersion>
     </PropertyGroup>
